ui/app/src/components/GraphiQL/GraphiQL.tsx

```diff
@@ -104,6 +104,7 @@ function GraphiQL(props: GraphiQLProps) {
     window.localStorage.setItem(`${storageKey}graphiql:query`, newQuery);
   };
   const explorer = explorerPlugin({
+    // @ts-ignore - Edit props does exist in GraphiQLExplorerPluginProps
```

Comment:
What does this mean?

Reply:
I made a typo, updated to `onEdit prop exists in GraphiQLExplorerPluginProps`. There's an error saying that the `onEdit` prop doesn't exist in `GraphiQLExplorerPluginProps`, which is incorrect. But there's no types package for GraphiQL to update

Reply:
Removed the comment and `editProp`. In newer version the prop does not exist anymore, and it uses the same `onEditQuery` prop that `GraphiQLComponent` has